spec/soft_error_spec.rb

```diff
@@ -46,7 +46,8 @@ def self.new(a); 1; end
       }.to raise_error(StandardError)
       expect(T::Coerce[CustomTypeDoesNotRiaseHardError].new.from(1)).to eql(1)
 
-      if Gem.loaded_specs['sorbet-runtime'].version >= Gem::Version.new('0.4.4948')
+      sorbet_version = Gem.loaded_specs['sorbet-runtime'].version
+      if sorbet_version >= Gem::Version.new('0.4.4948') && sorbet_version < Gem::Version.new('0.5.0')
```

Comment:
Did you find any compatibility errors if allow `0.5`?
I'm using `Sorbet-Rails` which cannot be downgraded to use Sorbet at `0.4`

Reply:
Hi! In the newer versions, subclasses of `T::Struct` will replace incompatible values with a `nil` when creating a new instance (assuming sorbet errors are not raised). This is not the current sorbet-coerce behavior.

I put up https://github.com/chanzuckerberg/sorbet-coerce/pull/26 to support it via configuration.

Reply:
Alternatively, we can move up the sorbet version required (0.5+) I think new people likely will use Sorbet 0.5